tutorial03_traceable_layers/README.md

```diff
@@ -39,14 +39,14 @@ the sample here takes second approach: use a separate lib module to deal with va
 
 Build Instructions
 -------------------
-- import the sample into your android studio (2.2.0 beta and above)
+- import the sample into your android studio 3.0+
```

Comment:
`- After downloading the sample, [import the project into Android Studio](https://developer.android.com/studio/projects/create-project.html#ImportAProject)`

Reply:
They could also use the "Import Sample" feature, no? That may be simpler.

Reply:
No, it is not part of the importable sample repo.